app/controllers/users_controller.rb

```diff
@@ -15,7 +15,121 @@ def show
 	    @time = current_user.time
 	    @cuisine = current_user.cuisine
 		@calories = self.class.calc_calories(@gender, @weight, @height, @age, @exercise, @goal)
-		Recipe.find_in_api
+		
+		# @all_recipes = Recipe.find_in_api
+
+		@all_recipes = {'items' => 
+				[
+				{'day' => 1,
+			 	'slot' => 1,
+			 	'id' => 123456,
+			 	'title' => "Eggs Benedict",
+			 	'calories' => 200,
+			 	'readyminutes' => 45,
+				 'price' => 2
+				 },	
+				{'day' => 1,
+			 	'slot' => 2,
+			 	'id' => 123456,
+			 	'title' => "Sweet & Sour Soup",
+			 	'calories' => 340,
+			 	'readyminutes' => 50,
+				'price' => 3
+				},	
+				 {'day'=> 1,
+				 'slot' => 3,
+				 'id' => 123456,
+				 'title' => "Mushroom Risotto",
+				 'calories' => 200,
+				 'readyminutes' => 75,
+				 'price' => 8	
+				 },
+				 {'day'=> 2,
+				 'slot' => 1,
+				 'id' => 123456,
+				 'title' => "waffles",
+				 'calories' => 200,
+				 'readyminutes' => 45,
+				 'price' => 8	
+				 },
+				 {'day'=> 2,
+				 'slot' => 2,
+				 'id' => 123456,
+				 'title' => "salad",
+				 'calories' => 200,
+				 'readyminutes' => 45,
+				 'price' => 8	
+				 },
+				 {'day'=> 2,
+				 'slot' => 3,
+				 'id' => 123456,
+				 'title' => "burrito",
+				 'calories' => 200,
+				 'readyminutes' => 45,
+				 'price' => 8	
+				 },
+				 {'day'=> 3,
+				 'slot' => 1,
+				 'id' => 123456,
+				 'title' => "grapes",
+				 'calories' => 200,
+				 'readyminutes' => 45,
+				 'price' => 8
+				 },
+				 {'day'=> 3,
+				 'slot' => 2,
+				 'id' => 123456,
+				 'title' => "cherries",
+				 'calories' => 200,
+				 'readyminutes' => 45,
+				 'price' => 8	
+				 },
+				 {'day'=> 3,
+				 'slot' => 3,
+				 'id' => 123456,
+				 'title' => "milk",
+				 'calories' => 200,
+				 'readyminutes' => 45,
+				 'price' => 8	
+				 }
+			]
+		}
+
+		@daily_recipes = Hash.new()
+
+		# helper: display days instead of indexes
+		days = {1 => "Monday", 2 => "Tuesday", 3 => "Wednesday",
+			4 => "Thursday", 5 => "Friday", 6 => "Saturday", 7 => "Sunday"}
+
+		@all_recipes['items'].each do |recipe|
```

Comment:
Maybe we should move all of the daily recipe calculations to other functions in order for them to be more easily tested and just more simple.